File: programacao/python/respostas/selecao/11.py
vendas = float(input("Digite o total de vendas: "))

if vendas >= 50000.0:
    comissao = vendas * 0.12
elif vendas > 30000.0:
    comissao = vendas * 0.095
else:
    comissao = vendas * 0.07

print(f"Comissao: {comissao:.2f}")

File: programacao/c/respostas/selecao/11.c
#include <stdio.h>

int main(void) {
    float vendas, comissao;

    printf("Digite o total de vendas: ");
    scanf("%f", &vendas);

    if (vendas >= 50000.0) {
        comissao = vendas * 0.12;
    } else if (vendas > 30000.0) {
        comissao = vendas * 0.095;
    } else {
        comissao = vendas * 0.07;
    }

    printf("Comissao: %.2f\n", comissao);

    return 0;
}